wherehows-web/app/styles/base/_base.scss

```diff
@@ -11,13 +11,21 @@ html {
  * Apply padding to top and bottom to account for navigation and footer
  */
 body {
-  padding-top: $nav-min-height;
-  padding-bottom: $nav-min-height;
   background-color: set-color(white, base);
   overflow-y: scroll;
   overflow-x: hidden;
 }
 
+.app-container {
+  margin-top: $nav-min-height;
+  margin-bottom: $nav-min-height;
+  transition: margin 0.7s ease;
+
+  &.banner-alert-offset {
+    margin-top: $nav-min-height + 52px;
```

Comment:
What does 52 mean?

Reply:
Will add that as a variable, it's supposed to be the banner height.